packages/venia-ui/lib/components/Head/headProvider.js

```diff
@@ -10,7 +10,7 @@ const removeExistingTitleTags = titleTags => {
 const VeniaHeadProvider = props => {
     useEffect(() => {
         const titleTagsToRemove = document.getElementsByTagName('title');
-        removeExistingTitleTags([...titleTagsToRemove]);
+        removeExistingTitleTags(Array.from(titleTagsToRemove));
```

Comment:
This fixes an issue where we would infinitely error in Edge.

Reply:
We'll have to chat about this; we surely use the spread operator more than just this.

Reply:
Right, spread is allowed, just not spread on a `NodeList` :)